src/scripts.js

```diff
@@ -24,49 +29,77 @@ const parseData = (data) => {
   const travelersData = data[0].travelers;
   const tripEntries = data[1].trips;
   const destinationEntries = data[2].destinations;
-  
   loadPage([travelersData, tripEntries, destinationEntries])
```

Comment:
Does this work if you just pass in travelersData, tripEntries & destinationEntries outside of the enclosure of an array? If so, I believe you should be able to just declare those as parameters on line 35 and then pass them in directly on lines 36-38.